src/templates/ja/2020/methodology.html

```diff
@@ -240,7 +245,7 @@ <h3 id="webpagetest"><a href="#webpagetest" class="anchor-link">WebPageTest</a><
       </p>
 
       <p>
-        HTTP ArchiveのWebPageTestのプライベートインスタンスは、最新のパブリックバージョンと同期して維持され、<a href="https://github.com/HTTPArchive/legacy.httparchive.org/tree/master/custom_metrics">カスタムメトリクス</a>で強化されています。これらは、テストの最後に各ウェブサイトで評価されるJavaScriptのスニペットです。<a href="https://github.com/HTTPArchive/legacy.httparchive.org/blob/master/custom_metrics/almanac.js">almanac.js</a>カスタムメトリクスには、DOMの状態に依存するメトリクスなど、そうでなければ計算できないいくつかのメトリクスが含まれています。
+        HTTP ArchiveのWebPageTestのプライベートインスタンスは、最新のパブリックバージョンと同期して維持され、<a href="https://github.com/HTTPArchive/legacy.httparchive.org/tree/master/custom_metrics">カスタムメトリクス</a>で強化されています。これらは、テストの最後に各ウェブサイトで評価されるJavaScriptのスニペットです。多くのデータアナリストの<a href="https://github.com/HTTPArchive/legacy.httparchive.org/commits/master/custom_metrics">貢献</a>に感謝します、特に<a href="https://github.com/HTTPArchive/legacy.httparchive.org/pulls? q=is%3Apr+author%3ATiggerito+sort%3Acreated-asc">献身的な努力</a>をした<a href="{{ url_for('contributors', year=year, lang=lang, _anchor='Tiggerito')}}">Tony McCreathのおかげで</a>、Web Almanacの2020年版では、HTTPアーカイブのテストインフラストラクチャの機能が3,000行以上の新しいコードで大幅に拡張されました。
```

Comment:
There's a space in there that shouldn't be there:

```suggestion
        HTTP ArchiveのWebPageTestのプライベートインスタンスは、最新のパブリックバージョンと同期して維持され、<a href="https://github.com/HTTPArchive/legacy.httparchive.org/tree/master/custom_metrics">カスタムメトリクス</a>で強化されています。これらは、テストの最後に各ウェブサイトで評価されるJavaScriptのスニペットです。多くのデータアナリストの<a href="https://github.com/HTTPArchive/legacy.httparchive.org/commits/master/custom_metrics">貢献</a>に感謝します、特に<a href="https://github.com/HTTPArchive/legacy.httparchive.org/pulls?q=is%3Apr+author%3ATiggerito+sort%3Acreated-asc">献身的な努力</a>をした<a href="{{ url_for('contributors', year=year, lang=lang, _anchor='Tiggerito')}}">Tony McCreathのおかげで</a>、Web Almanacの2020年版では、HTTPアーカイブのテストインフラストラクチャの機能が3,000行以上の新しいコードで大幅に拡張されました。
```

Reply:
Thanks, I fix.